course/templates/course/course_base.html

```diff
@@ -81,33 +81,29 @@ <h4>{% trans "Calendar" %}</h4>
                 <div class="alert alert-default">
                     {% trans "You are browsing this course in a different language than is set in your preferences." %}
                     {% if user_course_data %}
-                        <div class="row">
-                            <div class="col-lg-4">
-                                <form action="{{ url_without_language }}">
-                                    <button type="submit" class="aplus-button--secondary aplus-button--md btn-block">
-                                        {% blocktrans with lang=user_language|language_name_translated %}
-                                        Browse the course in {{ lang }}
-                                        {% endblocktrans %}
-                                    </button>
-                                </form>
-                            </div>
-                            <div class="col-lg-4">
-                                <form action="{{ instance|url:'set-enrollment-language' }}" method="post">
-                                    {% csrf_token %}
-                                    <input name="language" type="hidden" value="{{ query_language }}" />
-                                    <button type="submit" class="aplus-button--secondary aplus-button--md btn-block">
-                                        {% blocktrans with lang=query_language|language_name_translated %}
-                                        Set {{ lang }} as course language
-                                        {% endblocktrans %}
-                                    </button>
-                                </form>
-                            </div>
+                        <div class="button-container">
```

Comment:
How does this look now? Additionally, `aplus-button-container` would be more defining name (or possibly `aplus-button__container` @mtthwhggns ?).

Reply:
From my point of view, this is part of the alert, not of the button. To me it wouldn't make much sense to namespace it as if it's used inside the buttons. Of course, the alert isn't our component. 

Since it's only used in one place, and we might choose to have a custom component to do the job of the alert in the future (which would probably have an "actions" block, which might take buttons or whatever else), I think having a class separate from the component like this should be OK.

Reply:
Check. Let's go with this and let's assume that none else will use `button-container` for anything else for now.

Reply:
I updated the look of the buttons to the PR description.

> Check. Let's go with this and let's assume that none else will use button-container for anything else for now.

I think it's ok even if someone would use that somewhere else because now it's nested under the `alert alert-default`.

Reply:
The problem isn't for thiese style definitions to apply somewhere else, but if someone adds definitions for this class without nesting, then those other definitions will apply here too. But we can assume that won't happen.